python/faiss.py

```diff
@@ -481,9 +481,33 @@ def index_cpu_to_gpu_multiple_py(resources, index, co=None):
 def index_cpu_to_all_gpus(index, co=None, ngpu=-1):
     if ngpu == -1:
         ngpu = get_num_gpus()
-    res = [StandardGpuResources() for i in range(ngpu)]
-    index2 = index_cpu_to_gpu_multiple_py(res, index, co)
-    return index2
+    res = [StandardGpuResources() for _ in range(ngpu)]
+    index_gpu = index_cpu_to_gpu_multiple_py(res, index, co)
+    return index_gpu
+
+
+def index_cpu_to_gpu_selected_py(gpus, resources, index, co=None):
```

Comment:
please merge this function with index_cpu_to_gpu_multiple_py or have index_cpu_to_gpu_multiple_py call this function to avoid code duplication.

Reply:
done